includes/fields/class-attachment-upload.php

```diff
@@ -352,6 +352,7 @@ public function render_attachment( $attachment ) {
 		// Render attachment.
 		if ( strpos( $attachment->get_mime_type(), 'image/' ) === 0 ) {
 			$output .= wp_get_attachment_image( $attachment->get_id(), 'thumbnail' );
+			$output .= '<a data-component="file-download" href="' . esc_url( $attachment->get_url() ) . '" target="_blank" title="' . esc_attr__( 'Download', 'hivepress' ) . '"><i class="hp-icon fa-solid fa fa-arrow-down"></i></a>';
```

Comment:
Please don't use JS handlers if not used in JS, it's also better not to target them via CSS, this mixes JS and CSS logic (we use BEM notation and CSS classes, sometimes ids as an exception in CSS). A possible solution is targeting this link via ">" or other selectors, or adding a unique CSS class based on the field class and double underscores. Also make sure that the CSS above is as optimized as possible, without affecting other elements.